enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/VolumePermissionEnricher.java

```diff
@@ -175,8 +177,9 @@ public void visit(PersistentVolumeClaimBuilder pvcBuilder) {
                 if (pvcBuilder.buildMetadata() == null) {
                     pvcBuilder.withNewMetadata().endMetadata();
                 }
-                if (!pvcBuilder.buildMetadata().getAnnotations().containsKey(VOLUME_STORAGE_CLASS_ANNOTATION)) {
-                    pvcBuilder.editMetadata().addToAnnotations(VOLUME_STORAGE_CLASS_ANNOTATION, "standard").endMetadata();
+                String storageClass = getConfig(Config.storageClass);
+                if (Strings.isNotBlank(storageClass) && !pvcBuilder.buildMetadata().getAnnotations().containsKey(VOLUME_STORAGE_CLASS_ANNOTATION)) {
```

Comment:
can then be a check on null, too. Would also then rename 'storageClass' to 'defaultStorageClass' as it is only used when not already set. Might be confusing otherwise.